### ISSUE TITLE
Stock portfolio account is not reflecting change when changing Account Name

### ISSUE THREAD

**Issue by sogun771** [moneymanagerex/moneymanagerex#5705]
MMEX version:
 - [ ] 1.6.0
 - [ ] 1.6.1
 - [ ] 1.6.2
 - [ ] 1.6.3 Beta
 - [x] 1.6.4-Beta.2 64-bit

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

I am not sure if that's a bug or by design but modifying the account name in a stock portfolio is not reflecting on the main account.
It is instead modified in the shared account and account view. 

![image](https://user-images.githubusercontent.com/55146359/221055210-a214bbba-f90f-4f0f-859f-81d8619e3e57.png)

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
- In a stock portfolio account - Edit account
- Change Account Name

The name is changed in the Account View but not on the account itself.

Expected result:

Would have expected to see the name changed as well on the Stock Portfolio account.

Actual result:

It will change the Account view Name and the Shared Account

**Comment by n-stein** [comment]
The name you are trying to edit showing next to the briefcase icon (bottom of your image) is the "Company Name" of the investment which you can change by going to the stock portfolio, right clicking on the investment and selecting "Edit Stock Investment". When you right click in the nav panel and select "Edit Account" you are actually editing the associated share account.

**Comment by n-stein** [comment]
I notice that editing the "Company Name" doesn't force a refresh of the nav panel, so the change only shows up once something else refreshes it. Should we reopen and address this here, or log in a new issue?

**Comment by tactilis** [comment]
>  Should we reopen and address this here, or track in a new issue?

I'd say new issue as this one here is just an answer to a question.
The new issue will be listed as a Fix in the Release Notes
